src/openzaak/components/documenten/admin.py

```diff
@@ -18,17 +18,11 @@ class EnkelvoudigInformatieObjectInline(admin.StackedInline):
     extra = 1
 
 
-def unlock(modeladmin, request, queryset):
-    queryset.update(lock='')
-
-
 @admin.register(EnkelvoudigInformatieObjectCanonical)
 class EnkelvoudigInformatieObjectCanonicalAdmin(PrivateMediaMixin, admin.ModelAdmin):
     list_display = ['__str__', 'get_not_lock_display']
-    list_display = ['__str__']
     inlines = [EnkelvoudigInformatieObjectInline, GebruiksrechtenInline]
     private_media_fields = ('inhoud',)
-    actions = [unlock]
```

Comment:
Why was this removed?

Reply:
That's a 1.1 feature, and we're implementing 1.0.0 RC, so let's add it back because in 1.0.0 it's still relevant.

Reply:
`unlock` action is restored